configs/s2ef/all-forceonly/forcenet/forceonly.yml

```diff
@@ -0,0 +1,44 @@
+trainer: forces
+
+dataset:
+  - src: /checkpoint/electrocatalysis/relaxations/features/struct_to_energy_forces/train/200k/
+  - src: /checkpoint/electrocatalysis/relaxations/features/struct_to_energy_forces/val/is_30000/
```

Comment:
Let's update these paths to be consistent with the other configs (and not FAIR-specific)

Reply:
Sure, for naming of the file: `configs/s2ef/all-forceonly/forcenet/forceonly.yml` sounds okay?

Reply:
`configs/s2ef/all/forcenet/fn_forceonly.yml` might be better and consistent with what we did for [dimenet++](https://github.com/Open-Catalyst-Project/ocp/blob/master/configs/s2ef/all/dimenet_plus_plus/dpp_forceonly.yml).

Reply:
Fixed